src/quick-tour/customer-journey.md

```diff
@@ -2,26 +2,26 @@
 title: Customer Journey
 ---
 
-## [Attract New Customers]({{ site.baseurl }}{% link quick-tour/attract-customers.md %})
+## Attract New Customers
 
-Magento includes SEO functionality out of the box. Improve your search ranking and attract the most visitors to your site.
+Magento includes SEO functionality out of the box. Improve your search ranking and [attract]({{ site.baseurl }}{% link quick-tour/attract-customers.md %}) the most visitors to your site.
 
-## [Engage Your Customers]({{ site.baseurl }}{% link quick-tour/engage.md %})
+## Engage Your Customers
 
-Design your site with prepared templates, or create a custom design with features that invite people to interact with your store.
+Design your site with prepared templates, or create a custom design with features that [invite]({{ site.baseurl }}{% link quick-tour/engage.md %}) people to interact with your store.
 
-## [Increase AOV]({{ site.baseurl }}{% link quick-tour/increase-average-order-value.md %})
+## Increase AOV
 
-Increase average order value with promotions and content that encourage your customers to shop more.
+[Increase average order value]({{ site.baseurl }}{% link quick-tour/increase-average-order-value.md %}) with promotions and content that encourage your customers to shop more.
 
-## [Moment of Purchase]({{ site.baseurl }}{% link quick-tour/moment-of-purchase.md %})
+## Moment of Purchase
 
```

Comment:
Change this linked text from "Give" to the "faster and easier way to check out" text.

Reply:
**done**